i18n/scripts/replace-localize.ts

```diff
@@ -0,0 +1,7 @@
+import { FindAndReplaceGlobPatterns } from '../utils'
+
+const regex = new RegExp(/(localize)(\()(\s*?)(['`"])(\s*)(.*?)(\s*)(\4)(,?)(\s*)(\))/g)
```

Comment:
Please keep these in a descriptive variable name

Reply:
Fixed, Thanks :pray: